src/validations.ts

```diff
@@ -15,6 +16,21 @@ export function validateEmail(
   return null;
 }
 
+export function validateName(
```

Comment:
Could you add some tests for this function?

Reply:
Please, add a test to verify if the name `Ñoño` works, the same for Chinese characters as you did in the screenshot.

Reply:
And a test to validate if `Andr*s` does not work, should not, right?

Reply:
The same for `+`, `@`, `:`, etc.

Reply:
ok i test